02-image-classification.Rmd

```diff
@@ -11,165 +11,190 @@ cat(author)
-
-Land cover is ... it is important because ....
-
-Examine this online global land cover map created using imagery from Sentinel-2. In the search bar navigate to the town of Lytton, BC and compare land cover between 2018 and 2022. https://livingatlas.arcgis.com/landcoverexplorer/
-
-##### Q1: Describe the major change in annual land cover shown in these maps. What year did the most significant changes occur? Do some research to determine the cause of this shift in land cover. {.unnumbered}
-
-##### Q2: Describe how you might expect spectral reflectance in the NIR, Red and SWIR wavelegthns to change for a pixel undergoing a transition from forest to range land (bare) vegetation. {.unnumbered} 
-
-##### Q3: In 150 words or less, describe another application for annual land cover data generated from satellite imagery. Include a reference to one peer-reviewed academic article. {.unnumbered}
-
-In this lab, we will use the following processing steps to create a land cover classification for the greater Vancouver area and assess its accuracy: 
-
-1. 
-2. 
-3. 
-4. 
+Data will be accessed through the Google Earth Engine code editor. There are thousands of geospatial datasets stored within the GEE catalog. Landsat, Sentinel-1 and many other Earth-observing imagery are included with various levels of pre-processing in order to expedite analyses. Land cover, climate and other relevant environmental datasets are also available. 
 
 ------------------------------------------------------------------------
 
-## Task 2: Calculating NDVI and Built Up Index  {.unnumbered}
-
-In this task, you will calculate two spectral indices relating to vegetation and urban land cover using Landsat surface reflectance imagery covering Greater Vancouver. In the next task, you will use these indices to perform a supervised classification and produce a map of land cover. 
+## Task 1: Getting started with Google Earth Engine {.unnumbered}
 
-Spectral indices are mathematical equations containing spectral reflectance values from two or more wavelengths used to highlight areas of spectral importance in an image. There are a wide variety of spectral indices used to highlight a variety of different land covers and image properties including burned Areas (Normalized Burn Ratio), urban/ built up areas (Normalized Difference Built-Up Index), and water (Normalized Difference Water Index) to name a few. The Normalized Difference Vegetation Index (NDVI) is a frequently used spectral index that takes advantage of the high near-infrared reflectance and high red absorption properties of healthy vegetation and is therefore often used to quantify vegetation in a remotely sensed multispectral image. 
+**Step 1:** It is reccomended that you use/register for a gmail account prior to applying for access to GEE. Once you have a gmail account, you need to sign-up for GEE using the following link: https://signup.earthengine.google.com/
```

Comment:
May need to indicate here how students can anonymize their gmail accounts to protect their privacy.